<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="mainStacked"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDB">
    <property name="title">
     <string>DB</string>
    </property>
    <addaction name="actionDbClear"/>
    <addaction name="actionDbClearAll"/>
   </widget>
   <widget class="QMenu" name="menuRile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuRile"/>
   <addaction name="menuDB"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDbClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionDbClearAll">
   <property name="text">
    <string>Clear All</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
